README-CN.md

```diff
@@ -65,35 +73,38 @@ NebulaGraph内核 1.x 与 2.x 数据格式、通信协议、客户端等均双
 
 ## 快速使用
 
-请查看[快速使用手册](https://docs.nebula-graph.com.cn/3.2.0/2.quick-start/1.quick-start-workflow/)，开始使用 **NebulaGraph**。
+您可以在[云上](https://docs.nebula-graph.com.cn/3.3.0/2.quick-start/1.quick-start-overview/#_2)或[本地](https://docs.nebula-graph.com.cn/3.3.0/2.quick-start/1.quick-start-overview/#_6)快速体验 **NebulaGraph**。
```

Comment:
We cannot guarantee the correctness of the links from the master branch. In addition, some links are from other branches instead of the master, so we still need to change the links manually. 

I want to express that since we need to change the links manually and guarantee correctness, why not update this file every time the new version is released?